plugins/out_stackdriver/stackdriver.h

```diff
@@ -46,6 +49,20 @@
 /* Default Resource type */
 #define FLB_SDS_RESOURCE_TYPE "global"
 
+/* Holds a resource label and its value */
+struct flb_std_label {
+    const char *label;
+    flb_sds_t value;
+};
+
+/* Holds all information about a resource */
+struct flb_std_resource {
+    const char *type;
+    struct flb_std_label labels[10];
+    size_t label_cnt;
```

Comment:
Note: you either need the `NULL` sentinel at the end of the array, or the `label_cnt`, but not both. The advantage of the sentinel is that you don't need to count the array elements and there's no chance that the count will be mismatched with the labels (e.g., after copy/pasting an existing resource into a new resource, adding a label, and forgetting to update the count).

Reply:
I am going to try again with the NULL sentinal. It seemed like it wasn't being respected when I tried to use it previously, and cause a nasty little problem when using sizes to set msgpack stuff.